tensorboard/plugins/projector/projector_plugin.py

```diff
@@ -260,7 +262,12 @@ def get_plugin_apps(self):
         TENSOR_ROUTE: self._serve_tensor,
         METADATA_ROUTE: self._serve_metadata,
         BOOKMARKS_ROUTE: self._serve_bookmarks,
-        SPRITE_IMAGE_ROUTE: self._serve_sprite_image
+        SPRITE_IMAGE_ROUTE: self._serve_sprite_image,
+        '/index.js':
+            functools.partial(self._serve_file, 'tf_projector_plugin/index.js'),
```

Comment:
(Also, oof—the fact that this `functools.partial` works is not obvious,
though in hindsight it can be mostly inferred from the fact that bound
methods work at all… no action required, I suppose.)

Reply:
Ack. Btw, I copied https://github.com/tensorflow/tensorboard/blob/052a8ea45e90804544d0291200ddea7ff85099f2/tensorboard/plugins/core/core_plugin.py#L91-L105. 🙈

Reply:
Okay, fair enough—:see_no_evil: indeed. :-)